db/migrate/20230629155452_add_devise_to_users.rb

```diff
@@ -0,0 +1,49 @@
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
+
+      ## Recoverable
+      t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      ## Rememberable
+      t.datetime :remember_created_at
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
```

Comment:
- [ ] It would be better to uncomment these to unable the full functionality of the confirmable in devise, as we can see below, we have error signing up. Kindly test the application before submitting another code review request
![Screenshot from 2023-06-29 19-47-29](https://github.com/BahirHakimy/Rails-Blog/assets/46388113/77957923-b83d-480f-9e53-8e5b97568917)